cdap-client/src/main/java/co/cask/cdap/client/ServiceClient.java

```diff
@@ -199,12 +199,12 @@ public String getAvailability(ServiceId service) throws IOException, Unauthentic
    * @throws NotFoundException @throws NotFoundException if the app or service could not be found
    * @throws IOException if a network error occurred
    * @throws UnauthenticatedException if the request is not authorized successfully in the gateway server
-   * @deprecated since 4.0.0. Please use {@link #getServiceURL(ProgramId)}  instead
    */
-  @Deprecated
```

Comment:
How do you like the change `set routeconfig service <app-name.program-name> [version <app-version>] <route-config>` ?